dev-aws/customer-billing/topics.tf

```diff
@@ -417,3 +417,31 @@ resource "kafka_topic" "dummy-placeholder" {
     "cleanup.policy"    = "delete"
   }
 }
+
+# this topic is used during the transition from exstream to bex bill
+# generation and contains AccountReadyToBeFulfilledEvents 
+resource "kafka_topic" "transition-exstream-fulfilment" {
```

Comment:
Can I change the existing resource names or will that delete and recreate the topics 🤔

Reply:
Indeed, if you change that it will delete & recreate.
You will need to use [moved blocks](https://developer.hashicorp.com/terraform/language/modules/develop/refactoring#moved-block-syntax) to tell it it was renamed